incubator/goldfish/values.yaml

```diff
@@ -0,0 +1,57 @@
+# Default values for goldfish.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 1
+image:
+  repository: quay.io/tuannvm/goldfish
```

Comment:
Make sure you provide a stable version in the future.

Reply:
fixed